dali/kernels/common/scatter_gather.h

```diff
@@ -77,16 +77,23 @@ class DLL_PUBLIC ScatterGatherGPU {
     }
   }
 
+  enum class CopyMethod {
+    Default = 0,  // Uses scatter-gather kernel, unless there are 2 or fewer single effective copy
+                  // ranges, in that cases cudaMemcpyAsync is used
+    MemcpyAlways = 1,  // Always use cudaMemcpyAsync
+    MemcpyNever = 2,   // Always use scatter-gather kernel
+  };
```

Comment:
I don't know which style should we use, CamelCase or PASCAL_CASE. Just putting a comment here to double check (I'm good with both)

Reply:
For enum class I think we are using this format quite extensively: `EnumClassType::EnumValue`